gateway/ln-gateway/src/lib.rs

```diff
@@ -203,7 +204,7 @@ impl Gateway {
         payload: ConnectFedPayload,
         route_hints: Vec<RouteHint>,
     ) -> Result<()> {
-        let connect: WsClientConnectInfo = serde_json::from_str(&payload.connect).map_err(|e| {
+        let connect = WsClientConnectInfo::from_str(&payload.connect).map_err(|e| {
```

Comment:
Ah right, fixup for #1660

Reply:
Yeap!

After fix, I can't replicate [CI failure reported by Justin](https://github.com/fedimint/fedimint/pull/1718#issuecomment-1442766981)